product_supplierinfo_import_by_barcode/models/product_supplierinfo_import_template.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Tecnativa - David Vidal
```

Comment:
would be 2024 or it is old code?
```suggestion
# Copyright 2024 Tecnativa - David Vidal
```

Reply:
Nah, just copy-pasted... :sweat_smile: fixed